package.json

```diff
@@ -16,7 +16,8 @@
   "license": "MIT",
   "dependencies": {
     "dotenv": "^8.2.0",
-    "googleapis": "^61.0.0"
+    "googleapis": "^61.0.0",
+    "moment-timezone": "^0.5.32"
```

Comment:
With the changes I propose above, it would not be necessary to pull in the additional dependency.